.shippable.yml

```diff
@@ -95,6 +95,7 @@ build:
     - _make PLATFORM=ls-ls1088ardb
     - _make PLATFORM=ls-ls2088ardb
     - _make PLATFORM=ls-lx2160ardb
+    - _make PLATFORM=ls-ls1028ardb
```

Comment:
Can you just move it up before the `ls-ls1088ardb`, so we keep them somewhat sorted in alphabetic order?